libs/cart/src/drivers/interfaces/cart-order-service.interface.ts

```diff
@@ -0,0 +1,19 @@
+import { InjectionToken } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { DaffCart } from '../../models/cart';
+import { DaffCartPaymentMethod } from '../../models/cart-payment';
+
+/**
+ * The interface responsible for managing a customer's cart.
+ */
+export interface DaffCartOrderServiceInterface<T extends DaffCart = DaffCart> {
+	/**
+	 * Place an order.
+	 */
+  placeOrder(id: T['id'], payment?: DaffCartPaymentMethod): Observable<T>;
```

Comment:
`DaffCartPaymentMethod` can be off the `DaffCart` interface, right?

Reply:
Yeah but it should actually be generic right?

Reply:
Yes, it should be.

Reply:
done